edgelet/iotedge/src/main.rs

```diff
@@ -237,6 +241,34 @@ fn run() -> Result<(), Error> {
                 ),
         )
         .subcommand(SubCommand::with_name("version").about("Show the version information"))
+        .subcommand(
+            SubCommand::with_name("cssbundle")
+                .about("Bundles troubleshooting information")
+                .arg(
+                    Arg::with_name("location")
+                        .help("Directory to store bundle")
+                        .long("location")
+                        .takes_value(true)
+                        .value_name("DIRECTORY")
+                        .default_value("."),
+                )
+                .arg(
+                    Arg::with_name("tail")
```

Comment:
Why do we want a `tail` option for this command? Shouldn't the `since` date be the only way?

Reply:
yea

Reply:
yea